include/odp/api/spec/packet_io.h

```diff
@@ -83,6 +83,11 @@ extern "C" {
  * Do not wait on packet input
  */
 
+/**
+ * @def ODP_PKTOUT_MAX_QUEUES
+ * Maximum number of output queues
```

Comment:
Highlight that this is the maximum that API supports. Some thing like this:
"Maximum number of packet output queues supported by the API. Use pktio capability to check maximum number of queues per interface."

Reply:
Fixed in V3.